docs/rfcs/002-archival_storage/README.md

```diff
@@ -142,18 +142,16 @@ Digitised content will be ingested using Goobi, which should provide the bag lay
 ```
 b22036593/
 |-- data
-|   |-- b22036593.xml      // mets file
-|  [|-- bb22036593_001.xml // multiple manifestations]
-|  [|-- bb22036593_002.xml // multiple manifestations]
+|   |-- b22036593.xml      // mets "anchor" file for multiple manifestation
```

Comment:
Also the only METS file where it's not a MM...

Reply:
changed the b number to a real MM